test/app-tap/datetime.test.lua

```diff
@@ -438,6 +441,25 @@ test:test("Simple tests for parser", function(test)
     test:ok(date.parse('1998', { format = '%Y', tzoffset = '+03:00' }) ==
             date.new({ year = 1998, tzoffset = 180 }))
 
+    -- Testcases with override timezone by setting tz.
+    -- Timezone is not specified in a parsed string.
+    test:ok(date.parse("1970-01-01T01:00:00", {tz = 'MSK'}) ==
+            date.new{year=1970, mon=1, day=1, hour=1, min=0, sec=0, tzoffset=180})
```

Comment:
It is better to use `()` for function calls (not just given table as argument).
Also, please add spaces surrounding `=` in the table.
See https://www.tarantool.io/en/doc/latest/contributing/lua_style_guide/.
Here and below.

P.S. Since there are already "good" tests with the correct code style (not only with the "bad"), it is better to add new tests with the "good" code style.

Reply:
regarding `()`: I would follow a style used in testcases above in the same testsuite, because `()` missed almost always when a single argument is passed to a function

regarding missed spaces: added missed spaces, because mostly in the test spaces are not missed.

```diff
diff --git a/test/app-tap/datetime.test.lua b/test/app-tap/datetime.test.lua
index 85b43201f6..b4a3fc3bcf 100755
--- a/test/app-tap/datetime.test.lua
+++ b/test/app-tap/datetime.test.lua
@@ -443,17 +443,25 @@ test:test("Simple tests for parser", function(test)
 
     -- Testcases with override timezone by setting tz.
     -- Timezone is not specified in a parsed string.
-    test:ok(date.parse("1970-01-01T01:00:00", {tz = 'MSK'}) ==
-            date.new{year=1970, mon=1, day=1, hour=1, min=0, sec=0, tzoffset=180})
+    test:ok(date.parse("1970-01-01T01:00:00", {tz = "MSK"}) ==
+            date.new{year = 1970, mon = 1, day = 1,
+                     hour = 1, min = 0, sec = 0, tzoffset=180,
+                    })
     -- Timezone is specified in a parsed string as a military timezone.
-    test:ok(date.parse("1970-01-01T01:00:00Z", {tz = 'Europe/Moscow'}) ==
-            date.new{year=1970, mon=1, day=1, hour=1, min=0, sec=0, tzoffset=0})
+    test:ok(date.parse("1970-01-01T01:00:00Z", {tz = "Europe/Moscow"}) ==
+            date.new{
+                year = 1970, mon = 1, day = 1,
+                hour = 1, min = 0, sec = 0, tzoffset = 0
+            })
     -- Timezone is specified in a parsed string as an offset.
-    test:ok(date.parse("1970-01-01T01:00:00+01:00", {tz = 'Asia/Omsk'}) ==
-            date.new{year=1970, mon=1, day=1, hour=1, min=0, sec=0, tzoffset=60})
+    test:ok(date.parse("1970-01-01T01:00:00+01:00", {tz = "Asia/Omsk"}) ==
+            date.new{
+                year = 1970, mon = 1, day = 1,
+                hour = 1, min = 0, sec = 0, tzoffset = 60,
+            })
     -- Timezone is not specified in a parsed string and format is passed.
-    test:ok(date.parse('1998-11-25', { format = '%Y-%m-%d', tz = 'MSK'}) ==
-            date.new({ year = 1998, month = 11, day = 25, tzoffset = 180 }))
+    test:ok(date.parse("1998-11-25", { format = "%Y-%m-%d", tz = "MSK" }) ==
+            date.new{year = 1998, month = 11, day = 25, tzoffset = 180})
 
     -- Both tz and tzoffset are passed.
     assert_raises(test, ambiguous_timezone, function()

```